### PR TITLE
(#1597) Bump cactoos-matchers to 0.25

### PR DESCRIPTION
**Pull request by victornoel**
For #1597.

### CODE REVIEW

File: src/test/java/org/cactoos/list/BehavesAsList.java [hunk 1]
```diff
@@ -64,12 +64,12 @@ public boolean matchesSafely(final List<E> list) {
         new Assertion<>(
             "must have an index for the sample item",
             list.indexOf(this.sample),
-            new Verifies<>(i -> i >= 0)
+            new Satisfies<>(i -> i >= 0)
```
Comment (andreoss) [inline src/test/java/org/cactoos/list/BehavesAsList.java:67]: @victornoel `new IsGreaterOrEqualThan(0)` ?

File: src/test/java/org/cactoos/list/BehavesAsList.java [hunk 2]
```diff
@@ -64,12 +64,12 @@ public boolean matchesSafely(final List<E> list) {
...
         ).affirm();
         new Assertion<>(
             "must have a last index for the sample item",
             list.lastIndexOf(this.sample),
-            new Verifies<>(i -> i >= 0)
+            new Satisfies<>(i -> i >= 0)
```
Comment (andreoss) [inline src/test/java/org/cactoos/list/BehavesAsList.java:72]: @victornoel `new IsGreaterOrEqualThan(0)` ?

File: src/test/java/org/cactoos/collection/BehavesAsCollection.java [hunk 3]
```diff
@@ -78,7 +78,7 @@ public boolean matchesSafely(final Collection<E> col) {
             "Size must be more than 0",
             col,
             new IsCollectionWithSize<>(
-                new Verifies<>(s -> s > 0)
+                new Satisfies<>(s -> s > 0)
```
Comment (andreoss) [inline src/test/java/org/cactoos/collection/BehavesAsCollection.java:81]: @victornoel Could be be `new IsGreaterThan(0)` instead?

**Review by andreoss — CHANGES_REQUESTED** [review]

File: src/test/java/org/cactoos/list/BehavesAsList.java (hunk 1 above)
Comment (victornoel) [reply src/test/java/org/cactoos/list/BehavesAsList.java:67]: @andreoss I don't think this exists, do you have something particular in mind?

**Comment by victornoel** [comment]
@andreoss see my answer above

**Comment by andreoss** [comment]
@victornoel Indeed. I was thinking about `Matchers.greaterThan`, but let's not use them

**Review by andreoss — APPROVED** [review]

**Comment by victornoel** [comment]
@rultor merge

**Comment by 0crat** [comment]
Job was finished in 21 hours, bonus for fast delivery is possible (see [§36](http://www.zerocracy.com/policy.html#36))

**Comment by 0crat** [comment]
@sereshqua[/z](https://www.0crat.com/u/sereshqua) please review this job completed by @andreoss[/z](https://www.0crat.com/u/andreoss), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by sereshqua** [comment]
@0crat quality good

**Pull request merged**